salt/states/x509.py

```diff
@@ -555,8 +555,14 @@ def certificate_managed(name,
         new_comp = new
 
     new_certificate = False
+    # Compare the day difference only if the dates are not fixed
```

Comment:
Again some docs please

Reply:
Documentation was added to reflect the 2 new kwargs together with an example usage.